src/code/z_skin_matrix.c

```diff
@@ -1,42 +1,734 @@
 #include <ultra64.h>
 #include <global.h>
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_skin_matrix/func_800A6E10.s")
+/**
+ * Multiplies a 4 component row vector [ src , 1 ] by the matrix mf and writes the resulting 4 components to xyzDest
+ * and wDest.
+ * \f[
+ *     \left[\texttt{xyzDest}, \texttt{wDest}\right] = \left[\texttt{src}, 1 \right] \cdot \left[mf\right]
+ * \f]
+ */
```

Comment:
think we talked about this in discord but you can def remove this stuff. we're gonna prioritize how it looks in the code, not doxygen

Reply:
I don't think this should be removed. It doesn't impact readability in code since there is readable comments above it. If the two were blended in some way it would make sense to prioritise code readability, but this is entirely separate. It's very helpful to have these for math-heavy functions as reading the maths straight out of the code can often be difficult and very time-consuming if the function is large.

Reply:
i personally see it as clutter, but if it makes things alot easier to understand and is worth it, then sure ignore what i said

Reply:
It definitely impacts readability to some extent just by being there. That being said could it be written in a way that is readable as raw text, ideally over a single line?

Reply:
Put it on one line and removed the \left and \right tags